victims/tests/test_data.py

```diff
@@ -30,3 +30,10 @@ def test_data_cases_property(mocker):
     assert len(cases) == 5
     cases[0].when == date(2018, 10, 8)
     cases[-1].when == date(2018, 10, 3)
+
+
+def test_data_cases_property_by_request():
+    data = Data()
+    loop = asyncio.get_event_loop()
+    cases = loop.run_until_complete(data.cases())
+    assert len(cases) == 3
```

Comment:
This is not advisable. It makes tests depends on external connection and availability, and it also depends on envvar settings (have you seen it's failing on Travis? We use the production stylesheet there, probably that's why it's failing).